### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by klaudia-klis**

### CODE REVIEW

**Review by asakevych — CHANGES_REQUESTED** [review]
Great job!! 🚀 
I think your solution can be even better if you use switch and group your functions: 
```
case SortType.Name:
case SortType.Surname:
   return (order === 'asc')
      ? copyOfStudents.sort((a, b) => a[sortBy].localeCompare(b[sortBy]))
      : copyOfStudents.sort((a, b) => b[sortBy].localeCompare(a[sortBy]));
```
you can also create some function like `calculateAvarage` not to repeat same logic twice

File: src/sortStudents.ts [hunk 1]
```diff
@@ -1,16 +1,62 @@
...
+
+    default:
+      return (order === 'asc')
+        ? copyOfStudents.sort(
+          (a, b) => calculateAverage(a.grades) - calculateAverage(b.grades),
+        )
+        : copyOfStudents.sort(
+          (a, b) => calculateAverage(b.grades) - calculateAverage(a.grades),
+        );
+  }
```
Comment (choeqq) [inline src/sortStudents.ts:61]: try to use sort method only once, you can have separate function for picking values to sort, only then pass them to sort method

**Review by choeqq — CHANGES_REQUESTED** [review]
Logic looks really good, I think you could just clean up your code, with function for picking values, instead of using sort method 10 times 🚀

**Review by mvjl000 — APPROVED** [review]
Well done! 🚀

**Pull request closed without merging**